internal/pkg/describe/service.go

```diff
@@ -202,6 +204,22 @@ func NewServiceDescriber(opt NewServiceConfig) (*ServiceDescriber, error) {
 	}, nil
 }
 
+// Platform returns the platform of the task definition. If it is nil, fill in default values.
+func (d *ServiceDescriber) Platform() (*awsecs.ContainerPlatform, error) {
```

Comment:
What is the difference between a `ServiceDescriber` and `ECSServiceDescriber` I find it so confusing 😖 

Do we need the method on both types?
Is there a way of renaming the types to bring clarity for the difference between the two?

Reply:
Done. Previously, `ServiceDescriber` was for App Runner workloads, and `ECSServiceDescriber` was for other types. I know have a "base" describer for shared elements and `ECSServiceDescriber` and `AppRunnerServiceDescriber`. Changed some other names because there were so many "service"s and "describer"s but I'm not wed to the specific names, so welcome suggested changes. 
Hopefully this refactor make the code clearer.